clients/drcachesim/tests/multi_indir.templatex

```diff
@@ -1,7 +1,7 @@
 Schedule stats tool results:
 Total counts:
            3 cores
-          27 threads: .*W[012].T872902.*W[012].T872902.*W[012].T872902.*
+          27 threads: .*W[012].T([12]000)?d51c6.*W[012].T([12]000)?d51c6.*W[012].T([12]000)?d51c6.*
```

Comment:
So now we have redundant information: the workload ordinal follows the W and also is present in the top bits of T?

For csod, will we always show W0 and have to look at the T to find the workload id? Do we have a test for the multi_indir in csod case?

For csod, could schedule_stats use the workload-id from T to print as W; and also remove the redundant workload-id part from T (for csod and non-csod)?

Reply:
I changed this back to how it was, with a stripping of the workload.